README.rst

```diff
@@ -27,12 +27,12 @@ The ``opencensus-ext-newrelic`` package provides Python exporters to plug into
 .. _OpenCensus: https://opencensus.io
 
 
-Requirements
+Installation
 ------------
 
 OpenCensus-Python version 0.7.x is required to use the exporters.
 
-Using the trace exporter
+Getting Started
```

Comment:
There's both a trace exporter and a stats exporter. The header is modified here only for the trace exporter. Perhaps we should make sub-categories for the trace exporter rather than deleting that header?